components/brave_wallet/browser/keyring_controller.cc

```diff
@@ -193,17 +193,30 @@ void KeyringController::SetPrefForKeyring(PrefService* prefs,
 }
 
 // static
-void KeyringController::SetAccountNameForKeyring(
+void KeyringController::SetAccountInfoForKeyring(
```

Comment:
This function is better to be named `SetAccountMetaForKeyring`, AccountInfo might be confusing because we have `mojom::AccountInfo` which is designed for external used
Please also update the comment section in the beginning of the file to indicate the newly added `account_address` in `account_metas`

Reply:
Fixed and rebased in 10bd683f95363d743b2fb72a09278f0b5c3bb92e